lib/booking/ui/pages/manager_page/list_booking.dart

```diff
@@ -35,8 +36,28 @@ class ListBooking extends HookConsumerWidget {
         ref.watch(confirmedBookingListProvider.notifier);
     final managerConfirmedBookingListNotifier =
         ref.watch(managerConfirmedBookingListProvider.notifier);
+    final selectedDaysNotifier = ref.watch(selectedDaysProvider.notifier);
 
     final toggle = useState(!canToggle);
+
+    void handleBooking(Booking booking) {
+      bookingNotifier.setBooking(booking);
+      final recurrent = booking.recurrenceRule != "";
+      if (recurrent) {
+        final allDays = ["MO", "TU", "WE", "TH", "FR", "SA", "SU"];
+        final recurrentDays = booking.recurrenceRule
```

Comment:
You are using this a lot of time, i think you can create a function to retrieve recurrent days, and you can do so by using https://pub.dev/packages/rrule

Reply:
I used the parseRRule from syncfusion calendar package as it was already used for the calendar to avoid adding a new package